CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 - Removed: async mode/option
 ([#107](https://github.com/postcss/postcss-import/pull/107))
+- Removed: bower is not currently supported, use custom resolver
+([#116](https://github.com/postcss/postcss-import/pull/116))
```

Comment:
bower support is also bower.json support, which is not simple with main field as array.